File: CS245/Lab2/KM.java
public class KM{
    public static void main (String[] args) {
        double number = Double.parseDouble(args[0]);
        String unit = args[1];
        double kilometers = number * 1.609;
        double miles = number / 1.609;
        if (unit.equals("m")) {
            System.out.println(kilometers + " km");
        }
        else if (unit.equals("k")) {
            System.out.println(miles + " miles");
        }
    }
}

File: CS245/Lab3/KM.go
package main
import (
	"fmt"
	"os"
)

func main() {
	var number float32 = os.Args[0];
        var unit string = os.Args[1];
        var kilometers float32 = number * 1.609;
        var miles float32 = number / 1.609;
        if (unit == "m") {
            fmt.Println(kilometers + " km"); //Don't know how to print two different types
        } else if (unit == "k") {
            fmt.Println(miles + " miles");// Don't know how to print two different types
        }
}